check-links/markdown-absolute-file-links.bash

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -euo pipefail; 
+
+if [[ -d ./docusaurus ]]; then
+    DOCUSAURUS=$(realpath ./docusaurus)
+else
+    echo "ERROR: ./docusaurus not found" >&2 
+    exit 1
+fi
+
+# filter for markdown files that contain relative links to other markdown files
+findMarkdownFiles(){
+    find "$1" \
+        -regextype posix-extended \
+        -regex ".*\.mdx?" \
+        -print0 \
+        | xargs -0r grep -lE '\]\(\.\./[^)]+\.mdx?\)'
```

Comment:
Flaw: this fails to find Md files that only contain links with anchor fragments

Reply:
It's probably best to stop filtering entirely for Md find all Md files and it's redundant to the grep pattern that extracts relative links.

Reply:
@gberl002 I found a few more after fixing a bug in the script that caused it to ignore Markdown files that contained only fragment/anchor links.